sync/http/middleware.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/beatlabs/patron/correlation"
 	"github.com/beatlabs/patron/log"
 	"github.com/beatlabs/patron/sync/http/auth"
-	"github.com/beatlabs/patron/trace"
+	trace_http "github.com/beatlabs/patron/trace/http"
```

Comment:
why this change ? did we have an import collision?

Reply:
The base trace package is not used anymore, as functionality has moved to trace/http, which collides with net/http.